tests/migration/online_migration/yast2_migration.pm

```diff
@@ -209,7 +209,12 @@ sub run {
     send_key "alt-p";    # focus on the item of possible migration targets
     assert_screen 'yast2-migration-target-list-selected', 60;
     send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 20, 3;
-    send_key "alt-n";
+
+    # On some platforms such as Aarch64 and x86_64, after sending alt-n keys,
+    # we need to wait a while for the next screen to show up.
+    send_key("alt-n", wait_screen_change => 1);
+    # wait_still_screen 3;
```

Comment:
Huajian,
I suggest remove this line.

Reply:
now verifying it https://openqa.nue.suse.com/tests/5973295, thanks